.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,28 +1,27 @@
 ---
-name: Feature request
-about: Suggest an idea or an end-end feature you'd like to enable using AdaptiveCards
-title: "[Platform][Broader Feature Request Category if applicable] Title"
-labels: Request
-assignees: 
+name: Feature Request
+about: Suggest a feature you'd like to enable using AdaptiveCards
+title: "[Feature Request] Feature Title Here"
+labels: Request Triage-Needed
+assignees: ''
 ---
 
-**What platform is your feature request related to? (Delete other platforms).**
+# What is the problem you are attempting to solve with this feature?
 
-- [ ] .NET HTML
-- [ ] .NET WPF
-- [ ] Android
-- [ ] iOS
-- [ ] JavaScript
-- [ ] UWP
+A clear and concise description of what problem you are trying to solve (ex: I'm always frustrated when...)
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+# A high-level summary of your proposed solution to the problem
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+Please provide a high-level summary of your proposal
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+# What alternatives or workarounds have you considered
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+A clear and concise description of any alternative solutions or workarounds you've considered
+
+# Cross-Platform Enablement
+
+Would your solution work on every platform currently supported by Adaptive Cards?
```

Comment:
I don't think that's something we need to ask people. It's something we need to determine given the proposed solution, and propose an alternative if we establish that solution wouldn't work across platforms.

Reply:
Fair point...my goal was to force folks to think about that before opening platform-specific asks, but that is something we can handle on a case-by-case basis if it comes up